src/main/lombok/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java

```diff
@@ -12,6 +12,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Supplier;
```

Comment:
I don't see where this import statement is used. There are also 10 other import statements which are apparently not used anymore in this class and would have to be removed as well.